### PR TITLE
Update pgo_nsys_converter.py NVTX kern sum report name

### PR DESCRIPTION
**Pull request by mgoldfarb-nvidia**
Report name has changed with latest nsys.

If older versions are installed `pgo_nsys_converter.py` will fall back to the old report name.

### CODE REVIEW

**Comment by shraiysh** [comment]
@mgoldfarb-nvidia can you please change this to get the `nvtx_sum` report instead of the `nvtx_kern_sum`? I am not sure what this option was called before nsys 2023.1, (maybe `nvtxsum`?). Can you also find it out and handle that like you handled both `nvtx_kern_sum` and `nvtxkernsum`?

We need this because memory operations are not included in the protobuf file - they should be. We have an incomplete profile because we are using the wrong report.

**Comment by mgoldfarb-nvidia** [comment]
> @mgoldfarb-nvidia can you please change this to get the `nvtx_sum` report instead of the `nvtx_kern_sum`? I am not sure what this option was called before nsys 2023.1, (maybe `nvtxsum`?). Can you also find it out and handle that like you handled both `nvtx_kern_sum` and `nvtxkernsum`?
> 
> We need this because memory operations are not included in the protobuf file - they should be. We have an incomplete profile because we are using the wrong report.

Updated using `nvtx_sum` and `nvtxsum` for older nsys releases.

**Comment by shraiysh** [comment]
How do we add reviewers to this PR @mgoldfarb-nvidia?

**Comment by mgoldfarb-nvidia** [comment]
> How do we add reviewers to this PR @mgoldfarb-nvidia?
@nouiz might be able to help get this some eyes from the maintainers.

File: jax/tools/pgo_nsys_converter.py [hunk 1]
```diff
@@ -38,7 +38,14 @@
   profile_folder = os.path.join(os.path.split(args.profile_path)[0], '')
 
   assert isinstance(nsys_path, str)
-  stats_command = [nsys_path, "stats", "--force-overwrite", "true", "--force-export", "true", "--report", "nvtxkernsum", f"{args.profile_path}", "-o", f"{args.pgle_output_path}"]
```
Comment (pschuh) [inline jax/tools/pgo_nsys_converter.py:41]: Looks like the old report name was nvtxkernsum, but now you're doing nvtx_sum vs nvtxsum. What is the implication of that change?

Comment (shraiysh) [reply jax/tools/pgo_nsys_converter.py:41]: nvtx_kern_sum only provides information about kernel times and does not provide information about memory operations like wrapped copy. With the recent changes in XLA making the requirements for PGO stricter with PGLE, it is an error if the protobuf file has missing instructions (wrapped copy, for example). By using nvtx_sum, we will ensure we get time profile for memory operations too, and this profile gets accepted by XLA.

**Review by pschuh — APPROVED** [review]

**Pull request merged**